packages/react-components/react-motions-preview/stories/CreateMotionComponent/MotionArrays.stories.md

```diff
@@ -0,0 +1,16 @@
+`createMotionComponent()` supports arrays of `AtomMotion` objects. This is useful when you want to animate properties with _different_ durations, easings, etc.
+
+```ts
+const FadeFastGrowSlow = createMotionComponent([
```

Comment:
I think examples that also involve synchronizing delays would be useful to users

Reply:
An important delay use case in the near future: we want a **collapse** motion that expands the height first, then starts the fade-in after a delay (there could be some overlap). The exit would fade out, then collapse the height. This is useful for adding/removing items from lists.

Reply:
@ling1726 Is there something particular that you would like to see as an example?

> An important delay use case in the near future: we want a collapse motion that expands the height first, then starts the fade-in after a delay (there could be some overlap). The exit would fade out, then collapse the height. This is useful for adding/removing items from lists.

@robertpenner well, it's what this PR unblocks 👍

Reply:
@layershifter Yes, of course. The use case I mentioned happens to answer your question to Ling.